### PR TITLE
Put CDN URL on allowUrls for only trigger module errors

### PR DESCRIPTION
**Pull request by PoulainMaxime**
We need to test it when the modification is on the CDN
https://docs.sentry.io/platforms/javascript/configuration/options/#allow-urls

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Does this still work when the app in running in production? I think the env var is defined at compilation time but is then unset.

**Review by Quetzacoalt91 — APPROVED** [review]

File: _dev/src/utils/Sentry.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ if (store.state.app.psxMktgWithGoogleOnProductionEnvironment) {
     Vue,
     // @ts-ignore
     dsn: global.psxMktgWithGoogleDsnSentry,
+    allowUrls: [
```
Comment (Quetzacoalt91) [inline _dev/src/utils/Sentry.ts:11]: looks like it is `whitelistUrls` in the [docs](https://docs.sentry.io/clients/javascript/config/) though

Comment (PoulainMaxime) [reply _dev/src/utils/Sentry.ts:11]: whitelistUrls is deprecated

Comment (Quetzacoalt91) [reply _dev/src/utils/Sentry.ts:11]: Where did you find this information?

Comment (PoulainMaxime) [reply _dev/src/utils/Sentry.ts:11]: https://docs.sentry.io/platforms/javascript/guides/vue/configuration/filtering/

**Pull request merged**